machine/netberg/netberg_common_rangeley/kernel/series

```diff
@@ -0,0 +1,4 @@
+driver-igb-1.patch
+driver-igb-2.patch
```

Comment:
The patch igb-1 is to add Broadcom phy bcm5461s support and patch igb-2 is the driver functions of bcm5461s. These two patches will be combined to a single patch.